src/Services/UserManagementService/UsersManagementService.DAL/DI/DependencyInjection.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using UsersManagementService.DAL.Interfaces;
+using UsersManagementService.DAL.Interceptors;
+using UsersManagementService.DAL.Repositories;
+using UsersManagementService.DAL.Context;
+
+namespace UsersManagementService.DAL.DI;
+
+public static class DependencyInjection
+{
+    public static IServiceCollection AddDAL(this IServiceCollection services, IConfiguration configuration)
+    {
+        var connectionString = configuration.GetConnectionString(nameof(UsersDbContext))
+            ?? throw new InvalidOperationException("Connection string nameof \"UsersDbContext\" not found.");
```

Comment:
InvalidOperationException is not suitable type of exception here
This part: "nameof \"UsersDbContext\" looks strange